boards/nucleo-f031k6/include/periph_conf.h

```diff
@@ -20,12 +20,13 @@
 #ifndef PERIPH_CONF_H
 #define PERIPH_CONF_H
 
-/* No HSE available on this board */
-#define CLOCK_HSE            (0U)
-
 /* Adjust PLL factors when PLL is clocked by HSI */
-#define CLOCK_PLL_PREDIV     (2)
-#define CLOCK_PLL_MUL        (12)
+#ifndef CONFIG_CLOCK_PLL_PREDIV
+#define CONFIG_CLOCK_PLL_PREDIV     (2)
+#endif
```

Comment:
The stm32f03x and stm32f05x have a fixed `/2` prediv when HSI is used to clock the PLL, is this taken into account and how?

Reply:
Yes and this is a bit hacky I admit. The value of 2 defined for nucleo-f042 and nucleo-f031 corresponds to this `/2`. In theory, we should not allow it to be configurable when HSI is used and only for these cpu lines. Because not all f0 have this setup.
The current state of this PR reflects what's already in master but I agree that properly handle it would be better.
For now, the user is allowed to configure that value and do forbidden things, this can be improved in a follow-up PR I believe.

Reply:
Can you add a comment and an issue?

Reply:
I think the comment is still missing here :)